tortoise/tortoise_test.go

```diff
@@ -1052,6 +1053,7 @@ func TestBaseBallotEvictedBlock(t *testing.T) {
 		tortoise.TallyVotes(ctx, last)
 		verified = tortoise.LatestComplete()
 		require.Equal(t, last.Sub(1), verified)
+
```

Comment:
i am surprised empty line at the end doesn't fail lint.

Reply:
likely because it is inside the for loop. i removed